rtmp-client/src/main/cpp/librtmp/rtmp.c

```diff
@@ -4975,12 +4975,11 @@ RTMP_Read(RTMP *r, char *buf, int size)
   /* can't continue */
 fail:
   switch (r->m_read.status) {
+  case RTMP_READ_ERROR:  /* corrupted stream, resume failed */
+      SetSockError(EINVAL);
```

Comment:
It would be clearer if this case explicitly returned `r->m_read.status`, rather than falling through. Otherwise the code looks suspiciously like there's just a missing break or return.

Aside: please do fix the indentation.

Reply:
Done.